website/docs/core-concepts/media.md

```diff
@@ -4,4 +4,21 @@ sidebar_position: 7
 
 # Media
 
-## Coming Soon!
+Media to be used across the space can be found here. Media uploaded from any other place across Dega is also listed here.
```

Comment:
Change this line to:

All the media files used across the space are available here. The media files used in the space are reusable across different posts, settings, etc.

Reply:
updated